services/ingestion/etl/inforegio/json/src/events/onParseJSON.js

```diff
@@ -18,7 +18,7 @@ export const handler = async (event, context, callback) => {
 
   // Was the lambda triggered correctly? Is the file extension supported? etc.
   if (!snsRecord || snsRecord.EventSource !== 'aws:sns') {
-    return callback('Bad record');
+    throw new Error('Bad record');
```

Comment:
Why `throw` here when everywhere else we use `return callback(...)`?

Reply:
The callback was breaking the tests, and then I noticed you have used https://github.com/ec-europa/eubfr-data-lake/blob/master/services/ingestion/etl/agri/csv/src/lib/sns.js#L9

I honestly don't know why exactly these 2 work in one occasion and not in another, the only difference from before is the async/await pattern.

Reply:
Suggestions merged ;)